### ISSUE TITLE
Convert Excel to Yaml security_functions empty

### ISSUE THREAD

**Issue by fredericnavarro** [intuitem/ciso-assistant-community#1253]
**Describe the bug**
I try to  create new framework.
For this I used convert_library.py
It's ok for tab type 
- requirements
- scores
- implementation_groups

For the tab : security_functions, the file yaml Don't included data 

For the another test, i used the file intuitem/doc-pol.xlsx
Same result, the file doesn't inclued the doc et ploicies

**The result**
                      urn: urn:intuitem:risk:library:doc-pol
                      locale: en
                      ref_id: doc-pol
                      name: Documents and policies
                      description: Typical documents and policies recommended by intuitem
                      copyright: "\xA9 intuitem - 2024"
                      version: 1
                      provider: intuitem
                      packager: intuitem
                      objects: {}

**To Reproduce**
1. Go to 'tools
2. launch commande  'python3 convert_library.py esinfo/doc-pol.xlsx'
3. No erreur, message seems to be ok
                  parsing esinfo/doc-pol.xlsx
                  parsing tab library_content
                  processing library content
                  parsing tab Documents
                  parsing tab Policies
                  generating esinfo/doc-pol.yaml

**Expected behavior**
file included information

**Screenshots**
no erreur

**Environment (please complete the following information):**
ubuntu 6.8.0-51-generic
Python 3.12.3 (main, Nov  6 2024, 18:32:19) [GCC 13.2.0] on linux

About CISO Assistant
Version: v1.9.31
build: d4a9fa56
Disk space: 19.7 GB
Used: 6.5 GB (32 %)

**Comment by eric-intuitem** [comment]
security_functions has been deprecated in favor of reference_controls a long time ago. I don't see any reference to security_functions in the current repo except in old migrations.

When converting the provided doc-pol.xlsx file, the result is OK.

tools % python convert_library.py intuitem/doc-pol.xlsx 
parsing intuitem/doc-pol.xlsx
parsing tab library_content
processing library content
parsing tab Documents
processing reference controls
parsing tab Procedures
processing reference controls
parsing tab Policies
processing reference controls
parsing tab Others
processing reference controls
generating intuitem/doc-pol.yaml

**Comment by fredericnavarro** [comment]
Hello Eric,

Thanks for the answer. It's work now with the new syntax

The problem now is, the librairy are not update with the process describe in the following link
https://intuitem.gitbook.io/ciso-assistant/deployment/updating-your-local-instance

Is it the right command to update the library (/tools) with the program ?

**Comment by ab-smith** [comment]
Hello @fredericnavarro ,
Have you tried this: https://intuitem.gitbook.io/ciso-assistant/deployment/upgrading-a-library ?

**Comment by fredericnavarro** [comment]
Hello @ab-smith 

Yes maybe the action will solved when the application and repository on the serveur is uptodate
But, before this action, how to update  the file on the server ?
with the command : git pull origin main ?
It's enough ?

**Comment by ab-smith** [comment]
yes, make sure to do a `git fetch` and `git pull` first, then you can upgrade your instance using the `update-ciso-assistant.sh` script (https://github.com/intuitem/ciso-assistant-community/blob/main/update-ciso-assistant.sh) after a backup of the `db` folder.

**Comment by fredericnavarro** [comment]
Thanks
